### DISCUSSION TITLE
Tutorial 1: Example output for `ip a`, `ping 8.8.8.8`, `ip route` and `tracepath`

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#48] [Show and tell]
![linux_terminal_network_basics](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/8bab3a95-3485-4403-9176-fca97fd8180b)

**Comment by nyameko** [comment]
Similarly the same commands can be run from your head node:
![head_node_basic_networking_commands](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/7710bd3e-7d87-492a-a618-eab775369350)

**Reply by n-zwane** [reply]
Team Technocrats SCC 2025
<img width="944" height="959" alt="part-1" src="https://github.com/user-attachments/assets/0e0dafcc-222b-4170-b82a-fd0a68c39b75" />
<img width="944" height="1072" alt="part-2" src="https://github.com/user-attachments/assets/b8a88e43-8570-4362-98e0-535e8db5ff63" />

**Comment by iamyou-uoymai** [comment]
![Q A](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/120018440/45121b36-6733-41ca-806a-0622cabf20af)

**Comment by Kohzmik** [comment]
![Picture1](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/138365090/777d08ef-5616-4c90-b024-b5671070f2d4)

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/138365090/67a364f7-8ae8-4b3d-bb1b-090ea12bec5e)

**Reply by Kohzmik** [reply]
CUT_Team

**Comment by MishkaSingh1** [comment]
**EEElite(UCT)**
![Screenshot 2024-07-08 195128](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/acebb349-f8fa-4a02-a5d3-42db3f11ee83)
<img width="371" alt="image" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/79d8d860-b2bf-4cb4-8683-c8a59f3c3736">

**Comment by Cyphers-UFH** [comment]
<img width="660" alt="Screenshot 2024-07-08 214937" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/802cb712-81bd-4240-97b9-800e558f38fa">

**Comment by JCheney20** [comment]
**HPC Wizards**

![Screenshot 2024-07-08 230355](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/64533742/97ef0d7e-0e74-4c29-b9c5-739f79267cfa)

**Reply by Sange-Lugalo** [reply]
Hypernet



![WhatsApp Image 2024-07-11 at 13 37 22](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175015840/02f37c9b-a956-4a72-bff0-890a779d91ae)

**Comment by Dc-Spears** [comment]
CPUT - DC Spears
![CPUT-ip-stuff2](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/b0e672e6-0e38-4fcc-9539-98b04993f5eb)

**Comment by Rosemary2301** [comment]
![ip_a_ping_8888_AlgorithmAvengers](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/58eaa1d8-bde1-4579-ae81-363a159ba3cb)
![trace_path_AlgorithmAvengers](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/9a26237f-8b36-48e3-972d-cbe9de1cd606)

Algorithm Avengers

**Reply by Rosemary2301** [reply]
<img width="635" alt="tracepath_algoAven" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/0635fde3-823f-4e1c-9224-3b12cbce7bc9">

**Comment by wendymathonsi12** [comment]
![1](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/170462962/eeb34f3b-f370-4619-aa59-de867830fbb2)
![2](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/170462962/e19e29ec-4c9a-4b09-a80f-b62a2ce9c5f9)

**Reply by wendymathonsi12** [reply]
Cybershield

**Comment by nexusWorkers** [comment]
Nexus Networkers


![Screenshot 2024-07-09 082617](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175092203/81bc2227-aac3-4165-a8a2-b11dac085096)
![Screenshot 2024-07-09 082549](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175092203/6cbd50a3-728e-4c2c-8db3-069c9f7ee9f6)

**Comment by Talhahpatelia** [comment]
## Wits-A
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/f99cc878-23e8-4f6e-97e4-2d7125a55eec)
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/27ac6cce-e1e6-49dd-823b-32a0bce98310)
----
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/bc98302d-3e6b-4e25-8daf-f7a28b4b9bec)
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/22e9fa6e-1eb7-4778-8795-1629d9b2be2c)

**Comment by CraftierLamb599** [comment]
# Wits
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/9f81af7f-f8ac-4cee-8ae9-9f76db27561a)
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/9712c833-c947-41fd-a2c8-a82a0e9bdaef)

**Comment by LordyKing1224** [comment]
**Tomorrowslegends**

![WhatsApp Image 2024-07-09 at 15 09 49_2ba8ccbb](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/171877484/a820c14a-0a7f-4b67-82a1-10573f732c77)
![WhatsApp Image 2024-07-09 at 15 09 49_4eaff805](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/171877484/ce44c48b-67d5-42a8-b089-dde6e42c1e36)

**Comment by mbhonichau** [comment]
WSU_Elite
![Screenshot 2024-07-09 172650](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/160317954/54f8bf7d-096e-42db-9a01-1a7a239cca15)

**Comment by SfisoMncube** [comment]
Khulisa IP-A
![Screenshot (234)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/6ac9f729-7bc4-4e44-8d25-8e0ceedca5e8)

**Comment by ADELAIDEMOGALE** [comment]
Khulisa Data
![Screenshot (689)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/59803573-ea8a-438a-ac4c-5e048a2f36fa)

**Comment by Rosemary2301** [comment]
Algorithm Avengers
<img width="635" alt="tracepath_algoAven" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/455ba6a0-3df5-4b88-895c-61c739b515a9">

**Comment by ndumezulu-ngwenya** [comment]
UNISA-SCT'24
tracepath
![tracepath](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/89075456/cc32eb71-7bfe-471d-b449-e1d33c87268b)
ip a
![IP a ](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/89075456/a1187a3e-4033-4c7d-b647-1232088105e6)
ip route
![ip route](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/89075456/aa44bb60-d64a-4897-a7e9-e00ceb5de246)
ping
![ping](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/89075456/27658696-153e-478f-8232-80c0d27fe244)

**Comment by SizweMarole-21389** [comment]
Cybershield
![pinghead](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175023186/1f954926-23d1-4dcc-854e-1b2bc9834601)
![pinghead2](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175023186/0c34b926-a637-4e69-b60c-76dcff713576)

**Comment by hoperiley0806** [comment]
**Parallel Pioneers**

Ip a
![WhatsApp Image 2024-07-12 at 02 55 04_0f8c9992](https://github.com/user-attachments/assets/5cb2231b-64f1-40c0-8a9b-0be8713962a5)

Ping
![WhatsApp Image 2024-07-12 at 02 56 16_2aa5d8d2](https://github.com/user-attachments/assets/752bacfd-9ccc-499a-b5de-cb41cd56bae9)

IP route
![WhatsApp Image 2024-07-12 at 02 56 56_d1981e28](https://github.com/user-attachments/assets/ba9776e8-8964-49ed-a242-d2d091ea5d91)

Tracepath
![WhatsApp Image 2024-07-12 at 02 58 49_a2d65ebe](https://github.com/user-attachments/assets/39d5df70-27ad-45ea-9df2-c3b8a2262a60)

**Comment by JedK721** [comment]
**Wits-i**

ip a
![Screenshot 2024-07-13 155804](https://github.com/user-attachments/assets/ece11f73-1983-4ac1-bb38-0ff446f2bd54)

ping 8.8.8.8
![Screenshot 2024-07-13 160010](https://github.com/user-attachments/assets/6c96f081-68b4-4e6b-94c2-e546bb39800c)

ip route
![Screenshot 2024-07-13 160700](https://github.com/user-attachments/assets/9b6e1851-9c5b-464a-ac15-e221368f1bd9)

tracepath
![Screenshot 2024-07-13 161100](https://github.com/user-attachments/assets/05d3404b-3eeb-46f2-b5d2-bc8089d3f42a)

**Comment by F1R3BLAZ3** [comment]
![Screenshot 2025-07-07 154745](https://github.com/user-attachments/assets/cf796b22-fa7b-4519-892c-b48924b0da7f)
Ctrl+Alt+Defeat

**Comment by makmak-alt** [comment]
![Screenshot 2025-07-07 154714](https://github.com/user-attachments/assets/b9e3d8d9-213b-4d58-a000-adf8daff77ea)

Ctrl+Alt+Defeat

**Comment by ruvarashecodes** [comment]
![routeprint](https://github.com/user-attachments/assets/865c5bc0-c3db-42c6-9176-5c5a67b87c73)
Ctrl+Alt+Defeat

**Comment by amesie** [comment]
![image](https://github.com/user-attachments/assets/08b5ba7e-7b46-4298-9d62-d601f82dbe74)
Ctrl+Alt+Defeat

**Comment by DCspearsv2** [comment]
DC Spears
![tut1](https://github.com/user-attachments/assets/3dfab745-40a2-4b8a-aa19-52c9d164e641)

**Comment by KamvaMj** [comment]
DC Daggers


![Screenshot (69)](https://github.com/user-attachments/assets/19779bc4-9f01-41fb-9006-e4a7955d7c21)
![Screenshot (70)](https://github.com/user-attachments/assets/df49a148-3cc0-48ec-ae7e-055b7729f6f8)

**Comment by Iloke-Alusala** [comment]
0V1RCL0CK1D

![ip_a_rocky_chpc](https://github.com/user-attachments/assets/6b38c5bc-56c1-43c1-a727-4f104822c011)
![ip_route_rocky_chpc](https://github.com/user-attachments/assets/3f9301f5-384d-43b7-865c-2a442f5a0e72)
![Image 2025-07-07 at 16 44](https://github.com/user-attachments/assets/ecbb67ea-bb3d-4c8c-875d-7f254465fa05)

**Comment by Abahle947** [comment]
TechTribe 

![Screenshot 2025-07-07 163354](https://github.com/user-attachments/assets/216b0771-b9c6-445d-b106-d44e88be584c)
![Screenshot 2025-07-07 163119](https://github.com/user-attachments/assets/406b73a6-e969-42b2-bc22-7717aeb02b50)
![Screenshot 2025-07-07 163413](https://github.com/user-attachments/assets/b6563e7b-8c12-4e36-bff3-d7e12148d768)
![Screenshot 2025-07-07 163424](https://github.com/user-attachments/assets/5888cc3f-7261-4285-80d4-28207b250854)
![Screenshot 2025-07-07 163336](https://github.com/user-attachments/assets/24cfd38d-d33e-4459-9934-88ec9715c6ad)
![PHOTO-2025-07-08-11-21-00](https://github.com/user-attachments/assets/2bf28485-599c-4895-8258-6af11f10198a)

**Comment by makmak-alt** [comment]
![Screenshot 2025-07-07 171356](https://github.com/user-attachments/assets/de42ae69-6320-47b5-ac30-ba6bee4a0248)

Ctrl+Alt+Defeat
:)

**Comment by MphephuSamuel** [comment]
computer champions

![image](https://github.com/user-attachments/assets/1e5fe263-0fde-41c7-98d8-6270a6142e4b)
![image](https://github.com/user-attachments/assets/722bf1fa-d4ec-4a08-b36b-80ace679c6dd)

**Comment by UNISA-SCT25** [comment]
![linux commands results](https://github.com/user-attachments/assets/38a2ad2f-0886-45fb-8c82-32fc72e1a925)

**Comment by mdvar2** [comment]
![Screenshot 2025-07-07 201633](https://github.com/user-attachments/assets/2bae3ad4-ac96-4add-8c9a-5ea792cc8d19)

#Byte_Force

**Comment by Swissisking21** [comment]
techno geeks

![Screenshot (277)](https://github.com/user-attachments/assets/09436874-5916-4cd9-aca3-e7647a6c8661)
![Screenshot (276)](https://github.com/user-attachments/assets/578ba56f-d2af-45ad-9acb-7d354efafab5)
![Screenshot (275)](https://github.com/user-attachments/assets/8d7c9e19-af20-449e-983c-44b929d35880)
![Screenshot (274)](https://github.com/user-attachments/assets/ceffec9c-7dc1-4da1-abdc-68afe5dfa4f8)

**Comment by MubeenD1** [comment]
Wits-1
![image](https://github.com/user-attachments/assets/df48408b-6539-4665-82c3-575e61738e2e)

**Comment by Kevin-Ebrahim** [comment]
![Screenshot 2025-07-09 210922](https://github.com/user-attachments/assets/c7946ad4-7e12-44be-8582-e1f2f86497db)
![Screenshot 2025-07-09 211047](https://github.com/user-attachments/assets/dc09917f-cb94-4d5b-bc5c-7ad4a2c01f14)
Wits-A

**Comment by CadmusKei** [comment]
Team Jargon (UWC) SCC 2026 | Running basic network commands
<img width="1920" height="1080" alt="Screenshot From 2026-07-06 14-39-58" src="https://github.com/user-attachments/assets/236c776a-6275-4c70-98db-249ea2348024" />

**Comment by saxs-14** [comment]
3ncryp+3d (UMP) SCC 2026 | Running basic Linux network commands
<img width="1918" height="1078" alt="Screenshot 2026-07-06 145951" src="https://github.com/user-attachments/assets/744b5778-340e-4fc3-be01-637a47af3499" />

**Comment by Slindokuhle19** [comment]
Unizulu ComputeX
<img width="1120" height="750" alt="Screenshot from 2026-07-06 15-24-51" src="https://github.com/user-attachments/assets/8b44ec10-8917-4ddb-b79f-f362ab48789c" />
<img width="1153" height="1048" alt="Screenshot from 2026-07-06 15-25-49" src="https://github.com/user-attachments/assets/5b830805-f044-4fa0-ad4c-b38376e84a78" />

**Comment by VerilM** [comment]
Wits-1: 4 Networkig commands

<img width="1146" height="677" alt="Screenshot from 2026-07-06 15-47-40" src="https://github.com/user-attachments/assets/3c22ac44-4eb3-4211-b075-04159ab5d949" />
<img width="1146" height="677" alt="Screenshot from 2026-07-06 15-45-03" src="https://github.com/user-attachments/assets/d366ecb1-c838-4ac1-8420-fe1cadadf14a" />

**Comment by K659** [comment]
Resilient Geeks (UL)
<img width="937" height="740" alt="Screenshot from 2026-07-06 16-24-45" src="https://github.com/user-attachments/assets/96319d45-302b-48cc-9bd4-829f4c349b7d" />

**Comment by UNISA-QC-QUEENS** [comment]
UNISA_QC_QUEENS
<img width="807" height="362" alt="ip-a-command" src="https://github.com/user-attachments/assets/6d5bb3ba-bab0-4809-ad42-9fbbde8da2e6" />
<img width="807" height="362" alt="ip-route-command" src="https://github.com/user-attachments/assets/d2a2b499-a607-49c5-895f-d1a1fc9dbd05" />
<img width="807" height="362" alt="pin-command" src="https://github.com/user-attachments/assets/03aa4d28-65d8-4990-8bbc-cc8a8872b8b2" />

**Comment by RoarkeBailey** [comment]
<img width="1917" height="428" alt="Shot5" src="https://github.com/user-attachments/assets/db89feb2-a1c8-4621-8eb2-adc90061cc73" />
<img width="1918" height="690" alt="Shot4" src="https://github.com/user-attachments/assets/7255de73-2097-4fc5-96eb-ff6835dd8fee" />
<img width="1918" height="1010" alt="Shot3" src="https://github.com/user-attachments/assets/6cf0100e-9148-4135-9e33-65c2b78aa938" />
<img width="1917" height="717" alt="Shot2" src="https://github.com/user-attachments/assets/32232142-056b-4e0e-a869-31fdc45cae39" />
<img width="1918" height="1007" alt="Shot1" src="https://github.com/user-attachments/assets/1a972b1e-2871-46a2-b812-80ab142e23c3" />

G3MIKNIGHTS

**Comment by Lihle0512** [comment]
ChaosCompilers
<img width="1102" height="885" alt="Screenshot from 2026-07-06 21-37-27" src="https://github.com/user-attachments/assets/6b3135bc-0f61-4056-b1cc-01577cf7b138" />
<img width="1097" height="385" alt="Screenshot from 2026-07-06 21-38-28" src="https://github.com/user-attachments/assets/a8b238c7-f960-420b-a0a6-f768ebc6960a" />

**Comment by Lihle0512** [comment]
ChaosCompilers

<img width="1368" height="577" alt="Screenshot from 2026-07-06 21-44-07" src="https://github.com/user-attachments/assets/f5f2ede8-4dcb-445f-9616-3b454bbeb024" />

**Comment by bytedebuggers-dotcom** [comment]
**Byte Debuggers**
<img width="1299" height="723" alt="Screenshot From 2026-07-06 22-10-52" src="https://github.com/user-attachments/assets/738511f3-09d6-4ebe-bedd-7c41402617ab" />

**Comment by ThapeloVundla** [comment]
DC Spears | ip a, ping, ip route and tracepath command outputs

<img width="1014" height="413" alt="Screenshot from 2026-07-07 11-57-34" src="https://github.com/user-attachments/assets/bff9e6b4-9a24-4826-99a5-ea007adc20e8" />

<img width="989" height="565" alt="Screenshot from 2026-07-07 11-58-32" src="https://github.com/user-attachments/assets/0f440d4b-6b87-4470-8ccd-fce9240d7114" />

**Comment by mabotenick97-cloud** [comment]
CyberOps
<img width="1078" height="301" alt="Screenshot from 2026-07-07 12-37-36" src="https://github.com/user-attachments/assets/8b7eda0d-1489-491f-84e3-c92317d58141" />
<img width="753" height="427" alt="Screenshot from 2026-07-07 12-39-15" src="https://github.com/user-attachments/assets/73911a4f-d778-40d9-82e2-074d2a4d65d9" />
<img width="865" height="115" alt="Screenshot from 2026-07-07 12-39-54" src="https://github.com/user-attachments/assets/d9ace708-f180-4bc0-8aea-ea2e1d5caf84" />
<img width="629" height="392" alt="Screenshot from 2026-07-07 12-40-30" src="https://github.com/user-attachments/assets/3e8fc379-4855-47cf-9127-9c6dbe5b12ba" />

**Comment by s225875411** [comment]
<img width="1458" height="687" alt="image" src="https://github.com/user-attachments/assets/8d554266-5877-4721-afa0-2a2192ffa869" />

MadibazBits

The image shows ip a, ping, ip route and tracepath

**Comment by majaha8** [comment]
<img width="1871" height="1009" alt="Screenshot from 2026-07-07 21-08-38" src="https://github.com/user-attachments/assets/2452f711-6f90-41a5-a831-6209586a0e3c" />
<img width="1870" height="1044" alt="Screenshot from 2026-07-07 21-09-12" src="https://github.com/user-attachments/assets/645ee8ac-fdf5-482c-b8c7-dbaa42981f71" />

**Comment by Leo-OB** [comment]
## DC Axes (CPUT) | SCC 2026
### Outputs for `ip a`, `ping 8.8.8.8`, `ip route` and `tracepath` commands:
<img width="1351" height="503" alt="image" src="https://github.com/user-attachments/assets/2fecca76-d345-4b2e-9e89-709ae4eb9803" />

<img width="829" height="274" alt="image" src="https://github.com/user-attachments/assets/57568996-15a7-48af-a051-4c4ce99870bc" />

Note: '-c 4' in the above image just controls amount of outputted packets 

<img width="1019" height="156" alt="image" src="https://github.com/user-attachments/assets/47162c95-309c-45e9-9b82-5092c0bb1b01" />

<img width="760" height="437" alt="image" src="https://github.com/user-attachments/assets/43f7a19b-f740-4d56-998a-c0ca8e497f5d" />

**Comment by JayJay-lab** [comment]
Sol_Plaatje A  SCC 2026         <img width="1920" height="1200" alt="image" src="https://github.com/user-attachments/assets/99fb8339-0c14-4905-99b0-d47e9c0b1348" />

**Comment by JayJay-lab** [comment]
Sol_Plaatje_A SCC 2026    
<img width="1920" height="1200" alt="image" src="https://github.com/user-attachments/assets/1b414aa0-36d2-4f82-96dd-6e06e669ab43" />

**Comment by switchsquad** [comment]
<img width="1907" height="1005" alt="image" src="https://github.com/user-attachments/assets/4b09f749-3a41-48e8-a378-098db2887724" />

Switch Squad (TUT)

**Comment by mpho21925** [comment]
Ampere Alliance | SCC 2026 
Outputs for ip a, ping 8.8.8.8, ip route and tracepath commands:
<img width="1920" height="1080" alt="Screenshot 2026-07-08 083549" src="https://github.com/user-attachments/assets/a2a45c0e-02cf-478f-a88a-be340b43afcc" />

**Comment by 111nation** [comment]
**UPtime**

<img width="896" height="831" alt="image" src="https://github.com/user-attachments/assets/b4ea701a-3b4c-4d3c-8f04-28c930fac42f" />
<img width="818" height="850" alt="image" src="https://github.com/user-attachments/assets/7f39fa76-5cce-4337-8cfa-87eb40b3cae8" />
<img width="902" height="352" alt="image" src="https://github.com/user-attachments/assets/595b4bb3-27ff-4a57-b686-bad42950e8bc" />
<img width="881" height="884" alt="image" src="https://github.com/user-attachments/assets/8ae5c991-f7c7-4b9d-9db5-e3c41ff42489" />

**Comment by CadmusKei** [comment]
jargon (UWC) | SCC 2026 | Basic Networking
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/744c894b-7898-4346-92a3-bc57e0406d51" />

**Comment by majaha8** [comment]
Team Nebula|SCC 2026
<img width="1853" height="961" alt="ip" src="https://github.com/user-attachments/assets/1bb7b8ff-3b77-474c-80f7-ed53cf72a809" />

**Comment by LufunoMiles** [comment]
Digital Dynamos  [output for `ip a`, `ping 8.8.8.8`, `ip route` and `tracepath`](https://github.com/chpc-tech-eval/scc/discussions/48#top)]   <img width="947" height="796" alt="tracepath 8 8 8 8" src="https://github.com/user-attachments/assets/cdccb37d-ba6b-48aa-b849-121ed278a513" />
<img width="947" height="510" alt="ping" src="https://github.com/user-attachments/assets/f31a7af1-04b5-4409-80aa-bd28e8a3df50" />
<img width="947" height="279" alt="IPa" src="https://github.com/user-attachments/assets/3e315727-1959-4aca-9ac3-ad92b6c05df5" />

**Reply by LufunoMiles** [reply]
<img width="947" height="510" alt="ping" src="https://github.com/user-attachments/assets/077818b5-5a8d-4477-9713-a5137763029c" />

**Reply by LufunoMiles** [reply]
<img width="947" height="237" alt="Screenshot from 2026-07-09 15-51-33" src="https://github.com/user-attachments/assets/4556f51f-c898-4a1a-805e-e51cd2a1fdc0" />

**Comment by saxs-14** [comment]
3ncrypt3d (UMP) 
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/c9df92b8-c476-4301-a7b9-f3f6b95a6741" />

**Reply by ShavirV** [reply]
UPTime (University of Pretoria)

<img width="895" height="847" alt="image" src="https://github.com/user-attachments/assets/c9b4c40a-2964-436e-aedb-a7a32c67a48e" />

**Comment by Leeu-Fusi** [comment]
Cluster Commanders| CUT
<img width="1879" height="207" alt="Screenshot from 2026-07-10 11-59-39" src="https://github.com/user-attachments/assets/69aeb49a-d4c3-463b-a601-0a5fb69562a2" />
<img width="1853" height="866" alt="Screenshot from 2026-07-10 12-03-07" src="https://github.com/user-attachments/assets/d15fb7db-2820-4809-aa85-4e93a20d220f" />

<img width="1859" height="668" alt="Screenshot from 2026-07-10 11-59-05" src="https://github.com/user-attachments/assets/67558220-b07a-44af-99a6-9c72d81cc6ed" />

<img width="1865" height="1053" alt="Screenshot from 2026-07-10 11-58-10" src="https://github.com/user-attachments/assets/15c1aeec-7746-4f1f-b1aa-fde54370bb1e" />

**Comment by MrLampies** [comment]
DC Spears

<img width="1014" height="413" alt="Screenshot from 2026-07-07 11-57-34" src="https://github.com/user-attachments/assets/ae6bcf80-2f32-4f7b-8234-b5741477c009" />
<img width="989" height="565" alt="Screenshot from 2026-07-07 11-58-32" src="https://github.com/user-attachments/assets/6ce9c927-91dc-45c2-9d62-33f87a245051" />

**Comment by rsabreadking** [comment]
wit-1 (2026)
<img width="908" height="782" alt="image" src="https://github.com/user-attachments/assets/aa80db4a-8453-40a6-8cf9-db6da64f9a8b" />

<img width="546" height="527" alt="image" src="https://github.com/user-attachments/assets/9a2f54cb-18dc-4249-a07f-496a757f73a2" />

**Comment by Slindokuhle19** [comment]
UnizuluComputeX
<img width="1153" height="1048" alt="Screenshot from 2026-07-06 15-25-49" src="https://github.com/user-attachments/assets/63b1fdf5-57d1-4075-8734-47bcb6f7f341" />
<img width="1120" height="750" alt="Screenshot from 2026-07-06 15-24-51" src="https://github.com/user-attachments/assets/485182ea-18a4-44db-be29-a017a79cb5b7" />
